### PR TITLE
fix(Anontpp): update domain to "goplay.pw"

### PR DESCRIPTION
**Pull request by lizsdswsd**
## Description 

goplay/anontpp changed their domain from "goplay.ml" to "goplay.pw"

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/A/Anontpp/metadata.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		"nl": "Anontpp is de thuisbasis van de anime-streamingdienst GoPlay. U kunt ononderbroken gratis shows in HD bekijken zonder advertenties.",
 		"vi_VN": "Anontpp là nơi để xem anime trực tuyến, GoPlay. Bạn có thể xem các chương trình miễn phí, không bị gián đoạn bởi các quảng cáo."
 	},
-	"url": "goplay.ml",
+	"url": "goplay.pw",
 	"version": "1.1.9",
```
Comment (theusaf) [inline websites/A/Anontpp/metadata.json:15]: Bump the version
```suggestion
	"version": "1.1.10",
```

**Review by theusaf — CHANGES_REQUESTED** [review]
Please also provide screenshots showing that the presence is working.

**Comment by lizsdswsd** [comment]
> Please also provide screenshots showing that the presence is working.

it doesnt seem to be showing on my profile...

**Comment by theusaf** [comment]
> it doesnt seem to be showing on my profile...

If it doesn't appear on your Discord profile, try the troubleshooting steps listed in the support channel on Discord. If those don't work, try logging the presence data variable and taking a screenshot of the console logs instead?

**Comment by lizsdswsd** [comment]
> > it doesnt seem to be showing on my profile...
> 
> If it doesn't appear on your Discord profile, try the troubleshooting steps listed in the support channel on Discord. If those don't work, try logging the presence data variable and taking a screenshot of the console logs instead?

how do i do the second part?

**Comment by theusaf** [comment]
> > > it doesnt seem to be showing on my profile...
> > 
> > 
> > If it doesn't appear on your Discord profile, try the troubleshooting steps listed in the support channel on Discord. If those don't work, try logging the presence data variable and taking a screenshot of the console logs instead?
> 
> how do i do the second part?

Actually, it's okay. I've verified that the site's page content has not changed, so the presence should work.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**